### DISCUSSION TITLE
Is there a way to set `raw_user_meta_data` with `supabase.auth.signInWithOAuth`

### DISCUSSION THREAD

**Discussion by cristianbote** [supabase/supabase#29128] [Questions]
Hi, we're trying to adopt `supabase.auth.signInWithOAuth` specifically with google, and the OAuth flow works fine and users can register and login, but previously when we used  we had some important `meta_data` stored with:

```js
supabase.auth.signUp({
    email,
    password,
    options: {
        // Important meta data
        data: data
    }
});
```

My question is how we can still do that but with `signInWithOAuth` instead?

Thank you

**Comment by GaryAustin1** [comment]
There is not an option to do that.
This mentions why:
https://github.com/supabase/auth/issues/1056#issuecomment-1533102526

You would need to find an alternate way to get/save the user metadata.
